src/thunderbots/software/network_input/networking/network_client.cpp

```diff
@@ -177,6 +177,12 @@ void NetworkClient::filterAndPublishVisionData(SSL_WrapperPacket packet)
         world_msg.enemy_team = enemy_team_msg;
     }
 
+    if (Util::DynamicParameters::AI::refbox::override_refbox_defending_side.value() &&
+        Util::DynamicParameters::AI::refbox::defending_positive_side.value())
```

Comment:
Does "positive" have any meaning here? I'd almost rather "invert_field_side". 

Also, why do we need refbox overriding and the flag flipped?

Reply:
Positive does have meaning in that it refers to the "raw" field coordinates. I'd be tempted to leave the name as it is for now because it may link to how the refbox requests defending side.